.github/workflows/build-release-package.yml

```diff
@@ -98,7 +98,7 @@ jobs:
 
       - name: Create 7z Archive
         run: |
-          repo\Data\Tools\Precompiled\7za.exe a ezEngine.Release.${{ steps.sdk-version.outputs.content }}.7z .\_package\* -mx9
+          repo\Data\Tools\Precompiled\7z.exe a ezEngine.Release.${{ steps.sdk-version.outputs.content }}.7z .\_package\* -mx9
```

Comment:
I'm pretty sure that's not the file that we have checked in, no?

Reply:
7za.exe does not Support unpacking Nsis installers. This is needed for unpacking llvm though. So I replaced the 7za.exe with 7z.exe which does Support unpacking Nsis